axum/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
+- **fixed:** Include port number when parsing authority ([#2242])
```

Comment:
This should be in axum-extra's changelog, no? Also, it should mention that this affects the `Host` extractor, it's currently very hard to tell what this is about.

Reply:
true, the merge moved the files into axum-extra, but not the changelog :) good catch!

Reply:
https://github.com/tokio-rs/axum/pull/2242/commits/dfd438b933bf24f26f3c714231d43c8410989559